supervisor/homeassistant/core.py

```diff
@@ -392,7 +397,7 @@ async def _block_till_run(self, version: str) -> None:
                 break
 
         self._error_state = True
-        raise HomeAssistantError()
+        raise HomeAssistantCrashError()
```

Comment:
This needs an update on line 187 as well

Reply:
Nevermind  `HomeAssistantCrashError(HomeAssistantError)`

Reply:
Line should be 187 catch all exception from this function

Reply:
Yup, which is why I marked it as resolved :)